aapp_runner/tests/test_config.py

```diff
@@ -0,0 +1,519 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021 Adam.Dybbroe
+
+# Author(s):
+
+#   Adam.Dybbroe <a000680@c21856.ad.smhi.se>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for reading and manipulating configuration parameters
+"""
+
+import pytest
+from unittest.mock import patch, Mock, MagicMock
+import unittest
+import yaml
+from aapp_runner.read_aapp_config import VALID_CONFIGURATION
+from aapp_runner.read_aapp_config import check_config_file_options
+from aapp_runner.read_aapp_config import check_dir_permissions
+from aapp_runner.read_aapp_config import AappRunnerConfig
+from aapp_runner.read_aapp_config import (EnvironmentError, StaticConfigError,
+                                          ConfigFileOptionsError, AappWorkDirNotSet,
+                                          AappProcessKeyMissing, StationError)
+
+
+TEST_YAML_CONTENT_OK = """
+logging:
+  log_rotation_days: 1
+  log_rotation_backup: 30
+  logging_mode: DEBUG
+
+aapp_static_configuration:
+  decommutation_files:
+    hirs_file: hrsn.l1b
+    amsua_file: aman.l1b
+    amsub_file: ambn.l1b
+    mhs_file: ambn.l1b
+    avhrr_file: hrpt.l1b
+    msu_file: msun.l1b
```

Comment:
I realise this is just a test, but is this referring to the Microwave Sounding Unit from NOAA-14 and before?

Reply:
Ah, I guess maybe some users use aapp-runner for processing archive data.

Reply:
Your are right @gerritholl It is indeed a bit old now. So far I think we all use it for real-time only